lib/strategy.js

```diff
@@ -387,8 +386,12 @@ Strategy.prototype.configure = function(identifier, done) {
  * @return {Object}
  * @api protected
  */
-Strategy.prototype.authorizationParams = function(options) {
-  return {};
+AcpStrategy.prototype.authorizationParams = function(options) {
```

Comment:
I added the `params` object so we could build upon it and add other `options.param` to it when we want to support it. since it adds 2 simple lines, I thought it would be ok to add that and leave the door open for future edits.